xml/ns-System.Formats.Cbor.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Formats.Cbor">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types used in reading and writing data in the Concise Binary Object Representation (CBOR) format.</summary>
```

Comment:
@eiriktsarpalis Thanks for reviewing. I removed the spelled out acronym from all but reader and writer, but I also added a namespace summary here. Does this look okay?

Reply:
Here's the preview: https://review.docs.microsoft.com/en-us/dotnet/api/system.formats.cbor?view=dotnet-plat-ext-5.0&branch=pr-en-us-4992

Reply:
That's perfect, thanks!